### PR TITLE
Add OCD-ID's of Estonia

### PR DESCRIPTION
**Pull request by sguenther85**

### CODE REVIEW

File: identifiers/country-ee/top_level.csv [hunk 1]
```diff
@@ -1,2 +1,14 @@
 id,name
 ocd-division/country:ee,Estonia
+ocd-division/country:ee/ed:1,Haabersti and Põhja-Tallinn and Kristiine (districts of Tallinn)
+ocd-division/country:ee/ed:2,Kesklinn and Lasnamäe and Pirita (districts of Tallinn)
+ocd-division/country:ee/ed:3,Mustamäe and Nõmme (districts of Tallinn)
+ocd-division/country:ee/ed:4,Harju- (excl. Tallinn) and Raplamaa
+ocd-division/country:ee/ed:5,Hiiu- and Lääne- and Saaremaa
```
Comment (jloutsenhizer) [inline identifiers/country-ee/top_level.csv:7]: Is there any significance of the trailing hyphen on some of these names? It looks like it should be removed. I see them on the wikipedia page but https://www.riigiteataja.ee/en/eli/ee/514112013015/consolide/current doesn't have them.

**Review by jloutsenhizer — CHANGES_REQUESTED** [review]
The failures in the "Checks" tab is a new test that is checking if the compiled csv files match the expected output. Rerunning compile.py should correct it.

File: identifiers/country-ee/top_level.csv (hunk 1 above)
Comment (sguenther85) [reply identifiers/country-ee/top_level.csv]: No trailing hyphen anymore but the same issue.

**Comment by jloutsenhizer** [comment]
@jrunningen can you look into why the tests are failing? I ran them locally and did not see the same failures

**Comment by jrunningen** [comment]
It's because of the ordering. The compiler sorts the lines. Change `identifiers/country-ee.csv` to this:

```
id,name
ocd-division/country:ee,Estonia
ocd-division/country:ee/ed:1,Tallinn city districts of Haabersti and Põhja-Tallinn and Kristiine
ocd-division/country:ee/ed:10,The city of Tartu
ocd-division/country:ee/ed:11,Võru County and Valga County and Põlva County
ocd-division/country:ee/ed:12,Pärnu County
ocd-division/country:ee/ed:2,Tallinn city districts of Kesklinn and Lasnamäe and Pirita
ocd-division/country:ee/ed:3,Tallinn city districts of Mustamäe and Nõmme
ocd-division/country:ee/ed:4,Harju County (except Tallinn) and Rapla County
ocd-division/country:ee/ed:5,Hiiu County and Lääne County and Saare County
ocd-division/country:ee/ed:6,Lääne-Viru County
ocd-division/country:ee/ed:7,Ida-Viru County
ocd-division/country:ee/ed:8,Järva County and Viljandi County
ocd-division/country:ee/ed:9,Jõgeva County and Tartu County (except the city of Tartu)
```

Maybe the test should be made to disregard the ordering of rows, and also print a diff in the test log so it's easier to see the problem.

**Comment by jrunningen** [comment]
I've updated the test to ignore row ordering in #244.

**Comment by sguenther85** [comment]
Thanks @jrunningen for the help. That solved the problem.

So maybe now someone can check the whole thing for the correctness of the content :D.
@jpmckinney @azuser @jloutsenhizer

**Comment by azuser** [comment]
> Thanks @jrunningen for the help. That solved the problem.
> 
> So maybe now someone can check the whole thing for the correctness of the content :D.
> @jpmckinney @azuser @jloutsenhizer

Thanks ! It looks good to me.

**Review by jloutsenhizer — APPROVED** [review]

**Comment by jpmckinney** [comment]
I was quite surprised to see such long district names in the law! https://www.riigiteataja.ee/en/eli/ee/514112013015/consolide/current

**Pull request merged**